app/services/fact_importers/normalizers/katello.rb

```diff
@@ -0,0 +1,19 @@
+module FactImporters
+  module Normalizers
+    class Katello < Structured
+      def normalize
+        @facts = change_separator(@facts)
+        super
+      end
+
+      def change_separator(facts)
+        to_ret = {}
+        facts.each do |key, value|
+          to_ret[key.split('.').join(RhsmFactName::SEPARATOR)] = value
+        end
+        to_ret
```

Comment:
I always wonder about this, but is this better?
```suggestion
        facts.map do |key, value|
          [key.split('.').join(RhsmFactName::SEPARATOR), value]
        end.to_h
```

Or `Hash[facts.map { ... }]`

Also, split and join; should we instead use `key.gsub('.', RhsmFactName::SEPARATOR)`